syntax = "proto3";

package bookstore;

service FraudDetectionService {
    rpc DetectFraud (DetectFraudRequest) returns (DetectFraudResponse);
}

message DetectFraudRequest {
    string CreditCardNumber = 1;
    BillingAdress BillingAdress = 2;
}

message DetectFraudResponse {
    bool isLegit = 1;
}

message BillingAdress {
    string street = 1;
    string city = 2;
    string state = 3;
    string zip = 4;
    string country = 5;
}
